docs/sphinx/using/hardware.rst

```diff
@@ -0,0 +1,170 @@
+
+Setting Credentials
+```````````````````
+
+Programmers of CUDA Quantum may access the `IonQ Quantum Cloud
+<https://cloud.ionq.com/>`_ from either C++ or Python. Simply generate
+an API key from your `IonQ account <https://cloud.ionq.com/>`_ and export
+it as an environment variable:
+
+.. code:: bash
+
+  export IONQ_API_KEY="ionq_generated_api_key"
+
+
+C++
+```
+
+For developers in C++, you can indicate to ``nvq++`` that your quantum
+kernels will be executed in the IonQ Cloud via the ``--target`` flag.
+
+.. code:: bash
+
+    nvq++ --target ionq src.cpp ...
+
+This will take the API key and handle all authentication with, and submission to,
+the IonQ QPU.
+
+Note: A "target" in :code:`cudaq` refers to a quantum compute provider, such as :code:`ionq`.
+However, IonQ's documentation uses the term "target" to refer to specific QPU's themselves.
+
+Specifying the QPU
+'''''''''''''''''''
+
+At this time, programmer control over the IonQ QPU is under construction.
+
+
+Example
+.......
+
+.. literalinclude:: ../examples/cpp/ionq
+    :language: cpp
+
+
+
+Python
+```````
+
+For python developers, the target may be controlled with the ``cudaq::set_target()``
+function. This is functionally equivalent to the ``nvq++`` target,
+and will handle the submission of all quantum kernels to IonQ.
+
+.. code:: python
+
+    cudaq.set_target('ionq')
+
+Note: A "target" in :code:`cudaq` refers to a quantum compute provider, such as :code:`ionq`.
+However, IonQ's documentation uses the term "target" to refer to specific QPU's themselves.
+
+Specifying the QPU
+'''''''''''''''''''
+
+At this time, programmer control over the IonQ QPU is under construction.
+
+.. By default, the IonQ target will use the :code:`simulator` QPU.
+.. To specify which IonQ QPU to use, set the :code:`qpu` parameter.
+.. A list of available QPUs can be found `in the API documentation
+.. <https://docs.ionq.com/#tag/jobs>`_.
+
+.. .. code:: python
+..     cudaq.set_target("ionq", qpu="qpu.aria-1")
+
+Example
+........
+
+.. literalinclude:: ../examples/python/ionq
+   :language: python
+
+
+Quantinuum
+==================================
+
+Setting Credentials
+```````````````````
+
+Programmers of CUDA Quantum may access the Quantinuum API from either
+C++ or Python. Quantinuum requires a credential configuration file
+in your ``$HOME`` directory. This may be generated as follows, replacing
+the ``email`` and ``credentials`` in the first line with your Quantinuum
+account details.
+
+.. code:: bash
+
+  # You may need to run: `apt-get update && apt-get install curl jq`
+  curl -X POST -H "Content Type: application/json" -d '{ "email":"<your_alias>@email.com","password":"<your_password>" }' https://qapi.quantinuum.com/v1/login > $HOME/credentials.json
+  id_token=`cat $HOME/credentials.json | jq -r '."id-token"'`
+  refresh_token=`cat $HOME/credentials.json | jq -r '."refresh-token"'`
+  echo "key: $id_token" >> $HOME/.quantinuum_config
+  echo "time: 0" >> $HOME/.quantinuum_config
```

Comment:
The server helper seems to be doing some sort of refresh if the time is set to 0, which causes a print. Since the id token and the refresh token are freshly generated, what is the reason for setting time to 0 here?

Reply:
Errant `std::cout` left at `QuantinuumServerHelper.cpp:231`. 

I'll have a PR in a second to clean this up and to remove the need for the `time: 0` initialization.